backend/src/graphql/rooms.ts

```diff
@@ -0,0 +1,28 @@
+import gql from 'graphql-tag'
+
+export const createRoomMutation = () => {
+  return gql`
+    mutation (
+      $userId: ID!
+    ) {
+      CreateRoom(
+        userId: $userId
+      ) {
+        id
+      }
+    }
+  `
+}
```

Comment:
Why is this a function?

```suggestion
export const createRoomMutation = gql`
    mutation (
      $userId: ID!
    ) {
      CreateRoom(
        userId: $userId
      ) {
        id
      }
    }
  `
```